src/store/audio.ts

```diff
@@ -2060,17 +2063,84 @@ export const audioCommandStore = transformCommandStore(
             );
 
             // 読みの内容が変わっていなければテキストだけ変更
-            const isChangedMora = isAccentPhrasesTextDifferent(
+            const isSameText = !isAccentPhrasesTextDifferent(
               query.accentPhrases,
               accentPhrases
             );
+            let newAccentPhrases: AccentPhrase[] = [];
+            if (isSameText) {
+              newAccentPhrases = accentPhrases;
+            } else {
+              if (!state.experimentalSetting.shouldKeepTuningOnTextChange) {
+                newAccentPhrases = accentPhrases;
+              } else {
+                /*
+                 * # 調整結果の保持の仕組み
+                 * 1. 新しいAccentPhraseと古いAccentPhraseのテキスト（モーラのカタカナを結合したもの）を比較する。読点は無視する。（diffからflatDiff）
+                 * 例えば、
+                 * 旧：[ズ ン ダ モ ン ノ] [チョ ウ ショ ク]
+                 * 新：[ズ ン ダ モ ン ノ] [ユ ウ ショ ク]
+                 * という場合、
+                 *   [ズ ン ダ モ ン ノ]
+                 * + [ユ ウ ショ ク]
+                 * - [チョ ウ ショ ク]
+                 * のようなdiffが得られる。
```

Comment:
めちゃめちゃ細かいですけど、下で説明している「追加のdiff」と「削除のdiff」が未定義なのでここで説明してもいいかも？
```suggestion
                 * のような変更なしのdiff・追加のdiff・削除のdiffが得られる。
```